lib/src/schedule.dart

```diff
@@ -231,7 +231,19 @@ class DividerCardWidget extends StatelessWidget {
                   overflow: TextOverflow.ellipsis,
                 ),
                 const Divider(),
-                new Text(session.time.format(context), style: detailsStyle),
+                new Row(
+                  children: <Widget>[
+                    new Expanded(
+                      child: new Text(session.time.format(context), style: detailsStyle),
+                    ),
+                    new Text(
+                      session.duration == null || session.duration.inMinutes == 0
```

Comment:
To make this (and the check later) more terse, you could add a `isDurationNullOrZero` getter on `Session`.

Reply:
I thought about that but there are other changes I think should be made to session.duration to make it easier to have default durations, so I didn't want to polish the current implementation.